webpack/ForemanInventoryUpload/Components/PageHeader/components/PageDescription/PageDescription.js

```diff
@@ -10,14 +10,27 @@ export const PageDescription = () => (
     </p>
     <p>
       {__(
-        'In order to utilize these services, you can set the auto upload in the settings to "ON".'
+        'You can toggle the Auto upload switch to the ON position to enable Satellite to automatically upload your host inventory once a day.'
       )}
     </p>
     <p>
       {__(
-        'You can also trigger the upload manually by opening the relevant organization card, and clicking on the "Restart" button'
+        'Click Restart to upload your host inventory to Red Hat Insights. Perform this step for each organization from which you want to manually upload a host inventory.'
       )}
     </p>
+    <p>
+      {__(
+        'Enabling inventory uploads is required by subscription watch. For more information about subscription watch see link:'
+      )}
+      &nbsp;
+      <a
+        href="https://access.redhat.com/documentation/en-us/subscription_central/2020-04/html/getting_started_with_subscription_watch/assembly-about-subscriptionwatch"
+        target="_blank"
+        rel="noopener noreferrer"
+      >
+        About subscription watch
+      </a>
```

Comment:
for a follow-up, I think it would be awesome to have a new-tab-link helper/component as we use it quite a lot,
e.g: `<TabLink href={href}>{children}</TabLink>`